src/Functions/FunctionsConversion.h

```diff
@@ -306,6 +299,8 @@ struct ConvertImpl
                         {
                             if constexpr (std::is_same_v<ToDataType, DataTypeIPv4> && std::is_same_v<FromDataType, DataTypeUInt64>)
                                 vec_to[i] = static_cast<ToFieldType>(static_cast<IPv4::UnderlyingType>(vec_from[i]));
+                            else if constexpr (std::is_same_v<Name, NameToUnixTimestamp> && (std::is_same_v<FromDataType, DataTypeDate> || std::is_same_v<FromDataType, DataTypeDate32>))
+                                vec_to[i] = static_cast<ToFieldType>(vec_from[i] * 86400);
```

Comment:
It is not clear to the reader what this constant represents, please add a comment. Alternatively, if the constant is the product of seconds, minutes, etc., feel free to write the constant as product of factors.

Reply:
That's a great idea - I was surprised to not spot it in the code base.
I believe it would be appropriate to create a new relevant constant in the `src/Common/DateLUTImpl.h` file, what do you think?
And one more question - should it be a C-style macro `SECONDS_PER_DAY` (just like other definitions at the beginning of the header) or a `static constexpr Int32 seconds_per_day` member of the `DateLUTImpl` class, according to the common conventions of the ClickHouse code?

Reply:
Yepp, feel free to create a constant in DateLUTImpl.h if it is not there already. Be prepared for a scratch make after that as DateLUTImpl.h is #include'd everywhere :-)

A C++ style constant would be cleaner but consistency is king, so let's go with a C style constant. If needed, all macros could be converted to C++ constants in a separate PR.

Reply:
Done.
Decided to play it safe and opted for a good old C-style macro definition:
`#define DATE_SECONDS_PER_DAY 86400 /// Number of seconds in a day, 60 * 60 * 24`